kubernetes/bankapp-deployment.yml

```
@@ -1,63 +1,67 @@
       labels:
-        app: bankapp-deploy
+        app: bankapp
     spec:
       containers:
-      - name: bankapp
-        image: trainwithshubham/bankapp-eks:v2
+      - name: bank-pod
+        image: swayamnakshane/mybank:latest
         ports:
-        - containerPort: 8080
+          - containerPort: 8080
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)